coalib/results/SourceRange.py

```diff
@@ -10,7 +10,10 @@ class SourceRange(TextRange):
     @enforce_signature
     def __init__(self,
                  start: SourcePosition,
-                 end: (SourcePosition, None)=None):
```

Comment:
i don't change start, start would always be there right?, i don't provide any way in which the user could give `text, start_pos, end_pos` ONLY